api/api_gateway/api.py

```diff
@@ -814,17 +816,17 @@ def get_unsubscribe_link(subscription_id):
     return f"{BASE_URL}/unsubscribe/{subscription_id}"
 
 
-class ListImportPayload(BaseModel):
+class ListImportEmailPayload(BaseModel):
     list_id: UUID
     emails: conlist(EmailStr, min_items=1, max_items=10000)
```

Comment:
emails --- this will just be email: [data] coming from the app --- that way we can just use the phone / email type and not worry about plurals

Reply:
I'll make them both singular ... maybe change numbers to phone

Reply:
yup App is using phone